src/components/user/appointments/UserAppointments.js

```diff
@@ -6,12 +8,24 @@ import {
   StyledSearchForm, StyledSearchHeader, StyledSearchItems, StyledSearchList,
   StyledTitle,
 } from './UserAppointmets.styled';
+import SuccessIcon from '../../../assets/notification_success.svg';
+import { getAppointments } from '../../../store/user/patientOperations';
 import Appointment from './Appointment';
-import useRequest from '../../../hooks/useActions';
+import useActions from '../../../hooks/useActions';
+import { NotifySuccess } from '../../UI/Notify';
 
 const UserAppointments = () => {
-  const { appointments } = useRequest();
-
+  const [appointments, setAppointments] = useState([]);
+  const { isAdded, isAddedChangeState } = useActions();
+  useEffect(() => {
+    getAppointments(0, 100).then((response) => setAppointments(response.data.appointments));
+  }, []);
```

Comment:
Read about second param in useEffect - []

Reply:
In this case i fetch data for one time component render and i need no dependecies. If i understand clearly